community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/prettifier/PrettifierTest.scala

```diff
@@ -149,10 +149,18 @@ class PrettifierTest extends CypherFunSuite {
     )
   }
 
-  test("should handle call yield") {
+  test("should handle CALL YIELD") {
     actual("match (n) call db.indexes yield state RETURN *") should equal(expected("MATCH (n)%nCALL db.indexes YIELD state%nRETURN *"))
   }
 
+  test("MERGE should start on a new line") {
```

Comment:
You are right, but that test was wrongly named even before this PR. Let's not complect the two.